### ISSUE TITLE
How to build a parallel CNN using the same MLP in top with keras?

### ISSUE THREAD

**Issue by giovanniluccasilva** [fchollet/keras#1975]
Somethings like this:

model = Sequential()

model.add(Convolution2D(nb_filters1, nb_conv, nb_conv, border_mode='valid', input_shape=(1, img_rows, img_cols)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(nb_pool, nb_pool)))
model.add(Convolution2D(nb_filters2, nb_conv, nb_conv))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(nb_pool, nb_pool)))
model.add(Dropout(0.25))

model2 = Sequential()

model2.add(Convolution2D(nb_filters1, nb_conv, nb_conv, border_mode='valid', input_shape=(1, img_rows, img_cols)))
model2.add(Activation('relu'))
model2.add(MaxPooling2D(pool_size=(nb_pool, nb_pool)))
model2.add(Convolution2D(nb_filters2, nb_conv, nb_conv))
model2.add(Activation('relu'))
model2.add(MaxPooling2D(pool_size=(nb_pool, nb_pool)))
model2.add(Dropout(0.25))

model/model2.add(Flatten())
model/model2.add(Dense(512))
model/model2.add(Activation('softmax'))

Anybody can help me?

**Comment by giovanniluccasilva** [comment]
Please make sure that the boxes below are checked before you submit your issue. Thank you!

[+] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

[+] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

[?] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short). --> My model is pretty simple and the question is about general possibility of pretraining Embedding layer :)
